frontend/src/helpers/collaboration/socket_service.tsx

```diff
@@ -1,56 +1,78 @@
-"use strict"
```

Comment:
Let's not name this file with `.tsx` as there is no React component being used.

Reply:
Oops this was my bad.. 😓